.travis.yml

```diff
@@ -136,25 +146,31 @@ before_install:
     fi
   - python -V -V
   - python -c 'import sys; print("Python debug build:", hasattr(sys, "gettotalrefcount"))'
-  - travis_retry pip install --upgrade pip setuptools wheel
-  - travis_retry pip install cython
-  - if [ -n "$NUMPYSPEC" ]; then travis_retry pip install $NUMPYSPEC; fi
-  - travis_retry pip install --upgrade pytest pytest-xdist mpmath argparse Pillow codecov
+  - |
+    if [ "${TRAVIS_CPU_ARCH}" != "arm64" ]; then
+        travis_retry pip install --upgrade pip setuptools wheel
+        travis_retry pip install cython
+    fi
+  - |
+    if [[ "$NUMPYSPEC" && "$TRAVIS_CPU_ARCH" != "arm64" ]]; then travis_retry pip install $NUMPYSPEC; fi
+  - |
+    if [ "${TRAVIS_CPU_ARCH}" != "arm64" ]; then
+        travis_retry pip install --upgrade pytest pytest-xdist mpmath argparse Pillow codecov
```

Comment:
None of these packages have a dependency on `numpy`, so this can be moved into the `f [ "${TRAVIS_CPU_ARCH}" != "arm64" ];` above to make things less verbose